src/main/java/io/cdap/plugin/aws/s3/source/S3BatchSource.java

```diff
@@ -227,5 +246,50 @@ Map<String, String> getFilesystemProperties() {
       }
       return GSON.fromJson(fileSystemProperties, MAP_STRING_STRING_TYPE);
     }
+
+    public boolean shouldVerifyCredentials() {
+      try {
+        return this.verifyCredentialds;
+      } catch (Exception exception) {
+        LOG.info("Value received was {}. Default value for Verify Credentials i.e. False is used instead.",
+                 verifyCredentialds);
+        return Boolean.FALSE;
+      }
+    }
```

Comment:
We can do something like this here:

```
public boolean shouldVerifyCredentials() {
     return verifyCredentials == null ? false : verifyCredentials;   
 }
```

Reply:
Refactored the method. f778b8da21000ea82a97d829d58a1793faba0846